### PR TITLE
Create README.md for Google Assistant Add-On

### PR DESCRIPTION
**Pull request by mcarter14777**
Created ReadMe from Google assistant  integration page on home assistant website.

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @mcarter14777, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;hassio-addons#773).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by mcarter14777** [comment]
I know that these instructions are outdated but this will serve as a backbone for updating. I will try to configure this addon so I can actually update instructions

File: google_assistant/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# Hass.io Core Add-on: Google Assistant SDK
+
+<div class='note warning'>
```
Comment (frenck) [inline google_assistant/README.md]: These note/warnings work on our website, but will not work here on GitHub.

File: google_assistant/README.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+These instructions are outdated - the add-on has been updated and these are no longer accurate or complete.
+
+</div>
+
+<div class='note'>
```
Comment (frenck) [inline google_assistant/README.md]: Same as above

**Comment by mcarter14777** [comment]
I agree. I'll try to find a speaker, microphone and some spare time to try to work through the setup procedure and write some up to date documentation.

**Pull request closed without merging**